torch/ao/quantization/__init__.py

```diff
@@ -14,6 +14,7 @@
 from .stubs import *  # noqa: F403
 from .pt2e.eval_utils import _move_exported_model_to_eval as move_exported_model_to_eval
 from .pt2e.eval_utils import _move_exported_model_to_train as move_exported_model_to_train
+from .pt2e.eval_utils import _allow_exported_model_train_eval_for_special_ops as allow_exported_model_train_eval_for_special_ops
```

Comment:
nit: maybe just define these as public methods in `eval_utils`? not sure what is the difference between this and defining as public methods directly

Reply:
I kind of want to keep `eval_utils` a (semi) private namespace, since the file name is kind of TBD. Don't really want users to import from there directly. Do you think that's better than exposing this under top level toq?